### PR TITLE
new function to estimate a/b & a/c

### PR DESCRIPTION
**Pull request by bcarry**
Linked with issue #117

### CODE REVIEW

**Comment by bcarry** [comment]
I use black to reformat my code. Apparently it's not compliant with the tests here. Sorry
Which formater do you use?

File: fink_utils/sso/utils.py [hunk 1]
```diff
@@ -437,6 +437,38 @@ def compute_light_travel_correction(jd, d_obs):
...
+    # Taken at 2 sigma
+    amplitude = np.std(residuals) * 2.0
+
+    # Estimate the a/b ratio
+    a_b = 10 ** (0.4 * (amplitude * 2))
+
+    # Estimate the a/c ratio (and force c<b)
+    a_c = (a_b + 1) / (2 * R)
+    if a_c < a_b:
+        a_c = a_b
```
Comment (JulienPeloton) [inline fink_utils/sso/utils.py:467]: Just to make sure: if a/c < a/b, you want both to be equal (and not swap the two values)?

**Comment by JulienPeloton** [comment]
>  Which formater do you use?
Ruff :-) 
code source: https://docs.astral.sh/ruff/ (pip install ruff)
use in Fink: https://github.com/astrolabsoftware/fink-utils/blob/23ccf05dbbd1cdbedaac271ae964d54964a1a35c/.github/workflows/linter.yml#L28-L30

File: fink_utils/sso/utils.py (hunk 1 above)
Comment (bcarry) [reply fink_utils/sso/utils.py:467]: Yes: a/c is provided by the long-term behavior (kind similar to R in sHG1G2), while a/b is linked with the short term variability (rotation). But we expect a>b>c (principal rotator of a tri-axial ellipsoid).
As these are initial conditions, we set a/c = a/b if there is an issue with the "crude" estimate

Comment (JulienPeloton) [reply fink_utils/sso/utils.py:467]: Perfect :-)

**Pull request merged**